pytest_catchlog/plugin.py

```diff
@@ -50,12 +56,60 @@ def pytest_addoption(parser):
         dest='log_date_format', default=DEFAULT_LOG_DATE_FORMAT,
         help='log date format as used by the logging module.'
     )
+    add_option_ini(
```

Comment:
There already was [some discussion](https://github.com/eisensheng/pytest-catchlog/pull/33#issuecomment-177315532) about this.